.brigade/NOTES.txt

```diff
@@ -0,0 +1,35 @@
+This file was created by brig init and outlines next steps.
```

Comment:
Do we want to check this in (here and in the other repos with CI migrating from v1 to v2)?  It looks like this is the stock notes produced from `brig project init` -- maybe the intention was to keep it around for when we would want to add project-specific notes?

Reply:
I was iffy on keeping it or not. If you have a preference, I'm happy to go with it.

Reply:
Although it doesn't nec. harm anything with it being there, I might have a slight preference towards removing the file if it isn't intended to be updated w/ project-specific notes anytime soon.

Reply:
Np. I'm going to amend this PR and the other similar PRs to remove the notes.